src/BloomExe/Edit/EditingView.cs

```diff
@@ -727,13 +728,22 @@ private static bool CopyImageToClipboard(DomEventArgs ge, string bookFolderPath)
 				var path = Path.Combine(bookFolderPath, url.NotEncoded);
 				try
 				{
-					using(var image = RobustIO.PalasoImageFromFile(path))
+					using (var image = RobustIO.PalasoImageFromFile(path))
 					{
 						BloomClipboard.CopyImageToClipboard(image);
 					}
 					return true;
 				}
-				catch(Exception e)
+				catch (ExternalException e)
+				{
+					Logger.WriteEvent("CopyImageToClipboard -> ExternalException: " + e.Message);
```

Comment:
I suspect it doesn't work to put newlines in a string that will be localized; we won't have separate Windows and Linux tmx files. Instead, you'd need to get the localized string and *then* replace newlines with the one needed for this string.